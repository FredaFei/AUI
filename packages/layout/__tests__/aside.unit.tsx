import * as renderer from 'react-test-renderer'
import React from 'react'
// import {shallow} from 'enzyme'

import Aside from '../aside'

describe('测试 Aside 以下行为', () => {
  it('Aside 存在', () => {
    const json = renderer.create(<Aside>Aside</Aside>).toJSON()
    expect(json).toMatchSnapshot()
  })
  // it('Aside 设置 自定义类名', () => {
  //   const wrapper = shallow(<Aside className="test">Col-12</Aside>)
  //   expect(wrapper.find('.am-aside').hasClass('test')).toEqual(true)
  // })
  // it('Aside 设置 自定义样式', () => {
  //   const wrapper = shallow(<Aside style={{color: 'red'}}>Col-12</Aside>)
  //   expect(wrapper.render()).toMatchSnapshot()
  // });
})