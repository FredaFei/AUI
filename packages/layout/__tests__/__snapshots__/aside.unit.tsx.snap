// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 Aside 以下行为 Aside 存在 1`] = `
<div
  className="am-aside "
>
  Aside
</div>
`;
